input/resources/ValueSet-ehealth-sor-organization-specialty.json

```diff
@@ -29,6 +29,38 @@
             {
                 "system": "http://snomed.info/sct/554471000005108/version/20150731",
                 "concept": [
+                    {
```

Comment:
The added codes need to be added for system http://snomed.info/sct, not http://snomed.info/sct/554471000005108/version/20150731.